Woof/Woof/View/SitterFlow/SitterProfileView/SitterInfoCardView.swift

```diff
@@ -74,38 +74,21 @@ struct SitterInfoCardView: View {
     private let lineLimitForSurname = 1
     private let linelimitForPhoneNumber = 1
     private let linelimitForBio = 10
+    private let defaultCurrency = "$"
 }
 
 struct SitterInfoCardView_Previews: PreviewProvider {
     static var previews: some View {
+        let viewModel = SitterProfileViewModel()
```

Comment:
Should we integrate the view model into the view? Why do you use this view model separate from the view?